app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -53,6 +53,19 @@ class PocketVideoFetchScheduler(
             .setRequiredNetworkType(NetworkType.CONNECTED)
             .build()
 
+        val delay: Long
+        val workPolicy: ExistingWorkPolicy
+
+        // This sets the custom fetch delay for testing purposes.
+        // The custom delay will only run once.
+        if (IS_QA_BUILD) {
+            delay = DELAY_TIME_FOR_QA_MILLIS
+            workPolicy = ExistingWorkPolicy.REPLACE
+            IS_QA_BUILD = false
+        } else {
+            delay = getDelayUntilUpcomingNightFetchMillis(now, randLong)
+            workPolicy = ExistingWorkPolicy.KEEP
+        }
         // When the user foregrounds the app, we schedule a one time update for a random time inside our fetch interval
```

Comment:
nit: newline above. I generally always put a newline above a comment block, except if I'm starting an indent (e.g. for a function definition).